frontend/src/components/kanban/KanbanBoard.vue

```diff
@@ -22,9 +22,9 @@
             id="column"
           >
             <JobCard
-              v-for="job in this.getJobsByColumn[column.id]"
+              v-for="job in this.jobs[column.id]"
               :key="job.id"
-              :job="job"
+              :job="{ ...job }"
```

Comment:
Why duplicate the job?

Reply:
At one point that was the solution to the test printout issue, but looks like it might not be required anymore so will remove after testing